repository/src/server.rs

```diff
@@ -58,20 +58,27 @@ pub async fn run_server_legacy(path: &str, port: u16) -> GitServer {
     GitServer { child, daemon_pid }
 }
 
+pub enum PushVerifier {
+    AlwaysAccept,
+    AlwaysReject,
+    VerifierExecutable(String),
+}
+
 /// Runs a Simperby Git server with a push hook enabled.
 ///
 /// - `path` is the path to the root directory of a Simperby blockchain (not the repository path)
 /// - `port` is the port to run the server on
-/// - `simperby_executable_path` is the path to the Simperby executable, which will be executed by the hook.
-pub async fn run_server(path: &str, port: u16, simperby_executable_path: &str) -> GitServer {
+/// - `verifier_path` is the path to the verifier executable, which will be executed by the hook.
+///   If `None`, this server will unconditionally accept all pushes.
```

Comment:
What is this meaning? Does it mean `x` in `PushVerifier::VerifierExecutable(x)` is None?